### PR TITLE
Change to integer intervals, Use "IEdge" rather than "Edge"

### PR DESCRIPTION
**Pull request by hyanwong**
and rename  "TableGroup" to "Tables". We can refer to the tree sequence tables as `ts_tables` when used in a variable.

## Summary by CodeRabbit

- **Refactor**
  - Renamed key components to align with updated data structure terminology.
  - Updated attribute types for improved data integrity.

- **New Features**
  - Introduced a method for retrieving sample IDs, enhancing data accessibility.

- **Tests**
  - Adjusted test cases to reflect the refactored codebase and new naming conventions.

### CODE REVIEW

**Comment by hyanwong** [comment]
I also renamed "all_mutation_types_gig" to "all_sv_types_gig": I hope that makes sense @duncanMR (these aren't what we normally call mutations as present in the "mutations" table)

**Comment by duncanMR** [comment]
That's a good point re: mutations. Thanks for the changes!

File: GeneticInheritanceGraph/tables.py [hunk 1]
```diff
@@ -152,36 +163,104 @@ def append(self, obj) -> int:
...
 class NodeTable(BaseTable):
     RowClass = NodeTableRow
 
+    @property
+    def flags(self) -> npt.NDArray[np.uint32]:
+        return np.array([row.flags for row in self.data], dtype=np.uint32)
+
+    @property
+    def time(self) -> npt.NDArray[np.float64]:
+        return np.array([row.time for row in self.data], dtype=np.uint32)
```
[the remark this answers is not in the document]
Comment (hyanwong) [reply GeneticInheritanceGraph/tables.py:248]: Thanks! Well spotted

**Comment by hyanwong** [comment]
OK, merging. Fixes #43

**Pull request merged**